src/frameworks/next/index.ts

```diff
@@ -40,7 +40,11 @@ interface Manifest {
 }
 
 const CLI_COMMAND = join(
-  "node_modules",
+  require.resolve("next"), // ./dist/server/next.js (ref: https://github.com/vercel/next.js/blob/canary/packages/next/package.json)
```

Comment:
Could we instead get the NPM root somehow? `execSync("npm root", { cwd: ... })` making the assumption here is spooking me. Also we should add similar logic to the angular integration

Reply:
> Could we instead get the NPM root somehow?

I changed the following and re-tried "Scenarios Tested”. It looks fine.

https://github.com/firebase/firebase-tools/pull/5275/files#diff-372d171b067c11f7dbdea434bc98b44c356278e71e0f1251758e4ca25517c0b8R43

> Also we should add similar logic to the angular integration

I changed similarly vite and anular integrations. (5507397)

> Should add a changelog too

[TODO] I’ll update PR title and chagelog.md after changing policy is decided.